.firebaserc

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Should we add a cron job to clean up old OTPs?

Reply:
Right now the otp doc that is verified is deleted. I will change this to delete all otps for that email address once one is verified and in addition to that we can add a daily cron job to delete expired otps

Reply:
Old otps that are left in our db will happen when someone requests codes to multiple emails and does not verify any of them. We create one record per email (we override the record for multiple requests to the same email) and we delete the record once the user is verified and gets a token. We can add a daily cron job to handle expired otps older than a month or so IMO but they serve as a log for people trying to login and unable to get access, we can inspect the db and verify that we sent them a code - we did something similar for wallet

Reply:
Yeah, I was thinking about the expired ones. But it makes sense to leave it there for logging purpose.